VultisigApp/VultisigApp/View Models/EncryptedBackupViewModel.swift

```diff
@@ -227,16 +246,43 @@ class EncryptedBackupViewModel: ObservableObject {
         }
     }
     
+    func restoreVaultBak(modelContext: ModelContext,vaults: [Vault],vaultData: Data) {
+        do{
+            let vsVault = try VSVault(serializedData: vaultData)
```

Comment:
lets' move it into ProtoSerializer. The Idea is to work with Protobuf objects only inside ProtoMappable implementation.

Reply:
this one can't use ProtoSerializer , because it always base64 decoding and decompress , however , vault is not compressed

Reply:
you can extend ProtoSerializer